src/xo-app/index.js

```diff
@@ -1,54 +1,109 @@
 import React, { Component, PropTypes } from 'react'
-import { connect } from 'react-redux'
-import { Link, Route, IndexLink, IndexRoute } from 'react-router'
-import { ReduxRouter } from 'redux-router'
+import { connect, Provider } from 'react-redux'
+import store from '../store'
+import LoginForm from './login/loginForm.js'
+import VMForm from './vm/form.js'
+import { Router, Route, Link } from 'react-router'
+// from https://github.com/rackt/react-router/blob/master/upgrade-guides/v2.0.0.md#no-default-history
+import { hashHistory } from 'react-router'
 
-import About from './about'
-import Home from './home'
-import { actions } from '../store'
 
-let XoApp = class extends Component {
-  static propTypes = {
-    children: PropTypes.node,
-    counter: PropTypes.number
-  };
+const Dashboard = React.createClass({
```

Comment:
Avoid the functional syntax, use ES6 classes for stateful component.

We might (might!) use [preact](https://github.com/developit/preact/wiki/Differences-to-React) in the future.